core/annotations.c

```diff
@@ -352,12 +352,13 @@ instrument_annotation(dcontext_t *dcontext, IN OUT app_pc *start_pc,
 #    endif
```

Comment:
Update year range in Copyright notice, and elsewhere too.

Reply:
Done for all files changed here.